synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -44,6 +45,7 @@ export class SynopsysToolsParameter {
   }
 
   getFormattedCommandForPolaris(): string {
+    taskLib.debug(":::getFormattedCommandForPolaris:::");
```

Comment:
@sig-tithi @zaman-akib These debug logs are for us (synopsys-action's developers) only or for consumers as well. 

If these logs are for consumers as well, we should have user friendly messages.
Note - This comment applies for all debug logs.

Reply:
Removed this log as this is not needed for consumers